examples/gno.land/r/demo/boards2/post.gno

```diff
@@ -267,8 +298,13 @@ func (post *Post) Render(indent string, levels int) string {
 	if levels > 0 {
 		if post.replies.Size() > 0 {
 			post.replies.Iterate("", "", func(_ string, value interface{}) bool {
+				reply := value.(*Post)
+				if reply.isHidden {
+					return false
+				}
```

Comment:
For the moment is not an issue to filter out items inside the iteration function but is worth having in mind that if in the future we eventually use pagination we might have to remove the inner filter condition. Is ok to leave it like this until then.

Reply:
I'll add todo there